### DISCUSSION TITLE
Final Project - Knowledge Representation

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#29] [Homework]
Layout the topic you are interested in in a mind map:

- what are the main __themes__ of your project?
- what are the main __examples__ of your project?
- how are all those themes and examples __related__?

You can refer to the example we sketched around [Christianity](https://excalidraw.com/#json=pjH5P2kRISb0tcRMjR190,2gFHudGSe2OkdlfH9eRjRw).

**Comment by el-dah** [comment]
Not an exhaustive list by any means but here's a mind map we [+ Gabriel (@Gthunderr )] will continue expanding and referring to as we work on a site exploring culture of Berghain together

![image](https://github.com/user-attachments/assets/5a847a11-069a-4f45-a8dc-62b21586ffcf)

referencing my other post so I can keep track https://github.com/periode/augmenting-gallery/discussions/25#discussioncomment-11066822

**Reply by periode** [reply]
@eme9573 yes, groups should be done by next class. I'd also recommend emailing/contacting @Gthunderr directly, not everyone has Github notifications enabled!
